src/app/components/Editor.tsx

```
@@ -21,22 +22,109 @@ const Editor: React.FC<EditorProps> = ({ currentNote, updateNote }) => {
+      </div>
+      <div className="p-4">
+        <ReactMde
+          value={getContentWithoutTitle(currentNote.content)}
+          onChange={handleContentChange}
+          selectedTab={selectedTab}
+          onTabChange={setSelectedTab}
+          generateMarkdownPreview={(markdown: string): Promise<React.ReactNode> => {
+            const html = converter.makeHtml(markdown);
+            return Promise.resolve(
+              <div className="prose max-w-none">
+                <div dangerouslySetInnerHTML={{ __html: html }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)